scripts/i18n/upload.js

```diff
@@ -0,0 +1,68 @@
+const fs = require('fs');
```

Comment:
I'm not against this file (since I wrote it!) but I know @sampaccoud wanted to move off JS scripts and use Django management commands instead. Should we be adding this to another project when we plan on removing it from `richie`?

WDYT @sampaccoud @lunika ?

Reply:
While you were away, we discussed the fact that we want the container to include all the mechanics to upload/download/transform/compile translation files for both the front and the back. 
On the frontend it means that translations need to be taken out of the main build and to a separate .js file that we can rebuild upon deployment.
Not what is @lunika's latest plan :stuck_out_tongue:

Reply:
> On the frontend it means that translations need to be taken out of the main build and to a separate .js file that we can rebuild upon deployment.

I'm not sure I understand what you mean here. You want run a "fake build" that outputs to `/dev/null` to regenerate translations at deploy time?

I'm not sure how this affects the JS Script/Django command discussion. If we need the prod container to be able to run a JS build, it needs to include node anyway.

Reply:
my understanding was that it is a .js file containing only the translations that we just need to generate from the .po file.

Reply:
Oh ok, that you pull directly from `crowdin` at deploy time then.

That does not answer the initial question wrt. `upload.js` 😇

Reply:
I choosed to use the crowdin-cli :-) https://github.com/crowdin/crowdin-cli-2

Reply:
I did not use it the first time around because it's neither JS nor Python, but as you managed to use it through a container, this is pretty clean! Cheers!